.github/workflows/trunk.yml

```diff
@@ -270,3 +270,26 @@ jobs:
         PYTHON_EXECUTABLE=python ${CONDA_RUN} bash examples/models/llama2/install_requirements.sh
         # Test llama2
         PYTHON_EXECUTABLE=python ${CONDA_RUN} bash .ci/scripts/test_llama.sh stories110M.pt "${BUILD_TOOL}" "${DTYPE}" "${MODE}"
+
+  test-qnn-model:
+    name: test-qnn-model
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    strategy:
+      matrix:
+        dtype: [fp32]
+        model: [dl3]
+      fail-fast: false
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12-android
```

Comment:
@kirklandsign do we want to preinstall qnn for this docker image or have a dedicated docker image for qnn models?

Reply:
We don’t have that right now.

Reply:
@cccclai Moving the deps to docker can be done later. It shouldn't be a blocker for enabling qnn delegation in the benchmarking workflow